### PR TITLE
news: Cite GRASS with DOI - contributed by Peter Loewe

### PR DESCRIPTION
**Pull request by veroandreo**
Peter contributed this text snippet last year, but since the DOI was not working properly then I had this on hold. Thanks Peter!

Co-Authored-By: @ploewe

image

### CODE REVIEW

File: content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+like the current [GRASS GIS 8.3.2](https://doi.org/10.5281/zenodo.10817962), 
+or older versions going back to [GRASS GIS 7.8.6](<https://doi.org/10.5281/zenodo.5559460>).
+
+In addition, geospatial datasets for GRASS GIS can also be referenced by a DOI.
+See for example the Natural Earth GRASS dataset at <https://doi.org/10.5281/zenodo.3968936>.
+
+<a href="/images/news/grass_in_zenodo_citation.png">
+  <img src="/images/news/grass_in_zenodo_citation.png"
+   alt="GRASS GIS 8.3.2 in Zenodo"
+   title="GRASS GIS 8.3.2 in Zenodo"
```
Comment (neteler) [inline content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md:43]:
```suggestion
   alt="Citation of GRASS GIS 8.3.2 in Zenodo"
   title="Citation of GRASS GIS 8.3.2 in Zenodo"
```

File: content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   width="35%" style="float:right;padding-left:15px;padding-top:15px">
+</a>
+
+Since DOI can be interlinked by meta data, connected networks of GRASS GIS related 
+resources can be established, linking code, data and additional information, such as
+video recordings about GRASS GIS. Well known examples are the 
+GRASS code [evolution video](https://doi.org/10.5446/14652),
```
Comment (neteler) [inline content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md:50]:
```suggestion
GRASS GIS code [evolution video](https://doi.org/10.5446/14652), 
```

**Review by neteler — APPROVED** [review]

File: content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+[conference recordings](https://doi.org/10.5446/55251) or the historic
+[USACERL GRASS GIS commercial from 1987](https://doi.org/10.5446/12963),
+narrated by William Shatner of Star Trek fame.
+
+These DOI features are based on an underlying integration of the current 
+GRASS GIS code repository on GitHub with the Open Access repository 
+[Zenodo](https://zenodo.org/), which is also based on Open Source.
+
+A GRASS GIS DOI link will always resolve to a landing page within the Zenodo 
+repository, which is linked to the respective GutHub code repository. The
```
Comment (nilason) [inline content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md:60]: Typo `GutHub`, I just mention this in case someone can make a quick fix.

Comment (neteler) [reply content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md:60]: Thanks, fixed in #426

Comment (nilason) [reply content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md:60]: Great!

**Pull request merged**